### PR TITLE
 Fix #61: Omit Twitter errors

### LINKED ISSUE #61: Nothing works if Twitter widget cannot load
Screenshot 2019-02-20 at 16 15 10

### PR DESCRIPTION
**Pull request by 31z4**

### CODE REVIEW

File: src/components/ErrorBoundary.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ export default class ErrorBoundary extends React.PureComponent {
     }
 
     render() {
-        if (this.state.error) {
+        // Ignore errors related to Twitter widget.
+        if (this.state.error && !this.state.error.toString().includes('twttr')) {
```
Comment (summerisgone) [inline src/components/ErrorBoundary.js]: Here you're filtering errors by exception message, which is not quite correct, but might work.

File: src/api/GitHub.js [hunk 2]
```diff
@@ -373,7 +373,7 @@ export default class GitHub {
             throw e;
         }
 
-        return results.sort((a, b) => a.updated_at < b.updated_at);
+        return results.sort((a, b) => b.updated_at - a.updated_at);
```
Comment (summerisgone) [inline src/api/GitHub.js]: From the network you have strings: `updated_at: "2018-10-03T12:13:12Z"`. If you don't map them to date objects, neither comparison nor subtraction is correct. You should parse dates and compare them.

Comment (summerisgone) [reply src/api/GitHub.js]: image
subtraction however is less correct 😅

**Review by summerisgone — COMMENTED** [review]
Looks like you broke sorting

File: src/api/GitHub.js (hunk 2 above)
Comment (31z4) [reply src/api/GitHub.js]: Actually I fixed sorting :) All `update_at` here are date objects.

**Pull request merged**